modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -435,7 +435,11 @@ module.exports = {
             },
             {
               tag: 'img',
-              attributes: [ 'src' ]
+              attributes: [ 'src', 'alt' ]
+            },
+            {
+              tag: 'figcaption',
```

Comment:
Is this a bug fix that warrants mention in the changelog?

Reply:
Bit unclear if it was needed, I've removed this part to avoid side effect